.kokoro/release/publish_javadoc11.sh

```diff
@@ -67,7 +66,7 @@ for module in "${modules[@]}"; do
     echo "Running for ${NAME}-${VERSION}"
 
     # cloud RAD generation
-    mvn clean javadoc:aggregate -B -P docFX -DdocletPath=${KOKORO_GFILE_DIR}/${doclet_name}
+    mvn clean javadoc:aggregate -B -P docFX -DdocletPath=${KOKORO_GFILE_DIR}/${doclet_name} -Dcheckstyle.skip=true
```

Comment:
You may want to disable other irrelevant checks https://github.com/googleapis/google-cloud-java/blob/main/.github/workflows/ci.yaml#L46 has

```
        mvn -B -ntp \
            -Dclirr.skip=true \
            -Denforcer.skip=true \
            -Dcheckstyle.skip=true \
            -Dflatten.skip=true \
            -Danimal.sniffer.skip=true \
...
```